tools/camera_stream_validation/src/main.py

```diff
@@ -123,7 +133,12 @@
         else:
             logger.info("Output path: {} specified.".format(output_file))
 
-    input_file = json.load(open(args.input))
+    input_file = None
+
+    if args.secret:
```

Comment:
This seems to validate single camera each time, it's better we can batch the secret processing with the input of list of secrets we are interested.

Reply:
Thanks @abest0, this makes sense. I am good with this change.